### DISCUSSION TITLE
configure ping check in automation manager/checks

### DISCUSSION THREAD

**Discussion by hcimadomo-procsi** [amidaware/tacticalrmm#2096] [Q&A]
Hello,
in automation manager, in policy "server policy", i want add a ping check in checks . But the windows "edit ping check" ask an ip address !!
So f anyone have already configure  this, can you share your config please ?
Best regards

**Comment by wh1te909** [comment]
what exactly do you need help configuring? just put either a FQDN or IP address that you want to ping in that field.

**Comment by hcimadomo-procsi** [comment]
A policy is for many objects (in my case , all servers), so i cant put one address here. The logical parameter will be a variable that take the value of object checking.
Or i'm not understand the concept of policy ?
BR

**Reply by P6g9YHK6** [reply]
you can't put a variable on premade ping check
you need to script it.

but you are in your luck i made one with this exact feature.
https://github.com/amidaware/tacticalrmm/issues/2061#issuecomment-2464188291
